src/fr/yaoiscan/build.gradle

```
@@ -0,0 +1,10 @@
+ext {
+    extName = 'YaoiScan'
+    extClass = '.YaoiScan'
+    themePkg = 'mangathemesia'
+    baseUrl = 'https://yaoiscan.fr/'
```

Remove trailing slash
```suggestion
    baseUrl = 'https://yaoiscan.fr'
```